token/cli/src/main.rs

```diff
@@ -66,6 +66,7 @@ use sort::{is_supported_program, sort_and_parse_token_accounts};
 
 mod bench;
 use bench::*;
+use spl_token_2022::generic_token_account::GenericTokenAccount;
```

Comment:
micro-nit for next time: we typically prefer to keep the `use` statements together, so it would have been better to put this up at line 44-48 with the rest of the `spl_token_2022` imports

Reply:
Got it. Thank you.